proto/axelar/nexus/exported/v1beta1/types.proto

```diff
@@ -97,13 +103,12 @@ message GeneralMessage {
     STATUS_APPROVED = 1 [ (gogoproto.enumvalue_customname) = "Approved" ];
     STATUS_EXECUTED = 2 [ (gogoproto.enumvalue_customname) = "Executed" ];
   }
-
-  string id = 1 [ (gogoproto.customname) = "ID" ];
+  MessageID id = 1
```

Comment:
just to make sure, GeneralMessage was not yet included in a release, correct?

Reply:
yes, you are correct